admin/admin-base.php

```diff
@@ -111,46 +111,70 @@ public function init() {
 	}
 
 	/**
-	 * Adds the link to the languages panel in the WordPress admin menu
+	 * Adds links to Polylang's admin panels to the WordPress admin menu.
 	 *
 	 * @since 0.1
 	 *
 	 * @return void
 	 */
-	public function add_menus() {
+	public function add_menus(): void {
 		global $admin_page_hooks;
 
-		// Prepare the list of tabs
-		$tabs = array( 'lang' => __( 'Languages', 'polylang' ) );
+		$parent    = false;
+		$page_type = 'languages';
 
-		// Only if at least one language has been created
-		if ( $this->model->has_languages() ) {
-			$tabs['strings'] = __( 'Translations', 'polylang' );
-		}
+		foreach ( $this->get_sub_menu_items() as $tab => $tab_data ) {
+			$page = 'lang' === $tab ? 'mlang' : "mlang_$tab";
 
-		$tabs['settings'] = __( 'Settings', 'polylang' );
+			if ( empty( $parent ) ) {
+				$parent = $page;
+				add_menu_page( $tab_data['label'], __( 'Languages', 'polylang' ), $tab_data['capability'], $page, '__return_null', 'dashicons-translation' );
+				$admin_page_hooks[ $page ] = $page_type; // Hack to avoid the localization of the hook name. See: https://core.trac.wordpress.org/ticket/18857
```

Comment:
Could third parties used that "bug"?
My question being, if the hook was already internationalized before, maybe we should keep it that way for backward compatibility? Even though it's ugly as fuck...